core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -303,39 +307,40 @@ public void setMessageCorrelationID(javax.jms.Message message, String correlatio
 	}
 
 	public Destination getDestination() throws NamingException, JMSException, JmsException {
-		if (destination == null) {
-			String destinationName = getDestinationName();
-			if (StringUtils.isEmpty(destinationName)) {
-				throw new NamingException("no destinationName specified");
-			}
-			if (isLookupDestination()) {
-				if (!useTopicFunctions || getPersistent()) {
-					destination = getDestination(destinationName);
-				} else {
-					TopicSession session = null;
-					try {
-						session = (TopicSession) createSession();
-						destination = session.createTopic(destinationName);
-					} finally {
-						closeSession(session);
-					}
-				}
+		return getDestination(getDestinationName());
```

Comment:
Is het handig hier een != null check in te stoppen, en desnoods een exceptie terug te geven wanneer geen 'default' destination name gezet is?